Patches.py

```diff
@@ -93,6 +93,17 @@ def patch_rom(spoiler:Spoiler, world:World, rom:Rom):
     # Add it to the extended object table
     add_to_extended_object_table(rom, 0x195, keyring_obj_file)
 
+    # Load Key Ring model into a file
```

Comment:
```suggestion
    # Load Warp Song model into a file
```
Copy paste mistake ;)